api/schema/schema_validator.go

```diff
@@ -24,11 +24,12 @@ func NewSchemaValidator(t *testing.T, schemaName string, got interface{}) *Schem
 	if err != nil {
 		t.Fatal(err)
 	}
+	// Platform compatibility: use "/" separators always for file://
+	dir = strings.Replace(dir, "\\", "/", -1)
```

Comment:
I could surround this code with a `GOOS` check to not do it at all on non-Windows actually, will do that.